src/GraphQL/Language/CoreToVanillaConverter.cs

```diff
@@ -236,10 +236,13 @@ public IValue Value(GraphQLValue source)
 
                     // the idea is to see if there is a loss of accuracy of value
                     // for example, 12.1 or 12.11 is double but 12.10 is decimal
-                    double dbl = double.Parse(
+                    if (double.TryParse(
                         str.Value,
                         NumberStyles.AllowLeadingSign | NumberStyles.AllowDecimalPoint | NumberStyles.AllowExponent,
-                        CultureInfo.InvariantCulture);
+                        CultureInfo.InvariantCulture,
+                        out var dbl) == false) {
+                        dbl = str.Value.StartsWith("-") ? double.NegativeInfinity : double.PositiveInfinity;
+                    }
```

Comment:
Without this change, floating point numbers that overflow the maximum/minimum value will throw a format exception rather than some type of exception that inherits from `DocumentError`.  This change makes prior versions of .Net Core act the same as .Net Core 3.1, where overflowed values return negative or positive infinity.

Reply:
OK